src/app/screens/Receive/index.tsx

```diff
@@ -201,18 +200,13 @@ function Receive() {
           />
         }
       />
-      <div className="py-4">
-        <Container maxWidth="sm">
-          <div className={`${paid ? "bg-green-bitcoin" : ""}`}>
-            {invoice ? (
-              renderInvoice()
-            ) : (
-              <form
```

Comment:
Can we keep the `form` to keep the "submit on press enter" UX pattern?

Reply:
Ah, totally forgot about it! Thanks for reminding!